treeherder/webapp/api/performance_data.py

```diff
@@ -384,6 +386,11 @@ class Meta:
         ]
 
 
+class PerformanceTagViewSet(viewsets.ReadOnlyModelViewSet):
```

Comment:
This is doing its job well.
But for consistency reasons, it's best to sort these tags in the same manner we did with perf frameworks & issue trackers.
Otherwise, some JS excepts frameworks/issue trackers to be sorted, other JS expects tag to be unsorted, causing a bit more cognitive overhead.

Reply:
We can come back at a later time & remove backend sorting, if it's not the best idea.